app/views/tags/show.haml

```diff
@@ -10,6 +10,8 @@
 
 .container-fluid#tags_show
   .row
+    #left-navbar-fixed-sub-background.col-lg-1.visible-lg
```

Comment:
Classes should be listed before IDs (.col-lg-1 should precede #left-navbar-fixed-sub-background)